pkg/discovery/util/owner_util.go

```diff
@@ -19,13 +19,16 @@ type OwnerInfo struct {
 func GetOwnerInfo(owners []metav1.OwnerReference) (OwnerInfo, bool) {
```

Comment:
We may want to refactor the logic here a bit. Now we removed the continue statement when the IsController is false. In this case, the ownerSet var will always return true unless the owners is empty. 
Also, this logging below will be confusing too,
if ownerSet {
				glog.V(3).Infof("Multiple controller owners found: %+v", owner)
				continue
			}
In the case when you have 3 owners , the 2nd one is the controller one, and when you iterate to the 2nd one, ownerSet is set to true already and the log will be printed, which will be confused.

Reply:
Thanks @enlinxu for the diligence. Updated and improved the logging too.